server/models/TrustModel.js

```diff
@@ -7,22 +7,22 @@ class TrustModel{
   async get(){
     //const trust_relationship_instance = new trust_relationship(1);
     const list = await knex.select()
-      .table("trust_relationship");
```

Comment:
We don't have a "trust_relationship" table, but I think this references our "entity_trust" table.

Reply:
Yes, you are right, I just put the name here for a placeholder, because I just passed the unit test for this, not yet for the integration test, it is to say, I didn't finish the integration test, and run the app really.

Reply:
Oh okay, got it! I linked up the test to pass so we can add more features and write it out later after our unit tests.